WebScraper/seleniumNavigator.py

```diff
@@ -92,19 +93,17 @@ def run_scraper(term, output_to_json: bool = False, num_fails: int = 0):
             print("Max number of fails reached! Exiting...")
     else:
         print("Finished Successfully!")
-    finally:
```

Comment:
Why was this removed? I know its not necessary but it was nice for still getting output when errors occurred.

Reply:
Gonna add the FInally clause back. It won't work the same though. I removed it because I didn't like that it would output json even it it failed or was exited prematurely. Now, the finally case should only run if there are errors or interrupts

Reply:
Works for me.